app/tests/e2e/signin.spec.js

```diff
@@ -92,8 +92,8 @@ async function signOut(browser) {
 
 async function signIn(browser) {
   await openMenu(browser)
-  await browser.waitForElementVisible("#sign-in", 10000, true)
-  await browser.click("#sign-in")
+  await browser.waitForElementVisible(".trigger", 10000, true)
```

Comment:
this is not readable can we give  the elements an id? same for the next line

Reply:
the issue here for E2E is that one thing is what we see in the code and a very different what is actually rendered due to the popover element. 

Oh, but you meant something else. Ok, I see. Let's try. It is a bit confusing now with this popover thingy